python/datashaper/datashaper/types/workflow_options.py

```diff
@@ -0,0 +1,12 @@
+"""Data-types for workflow options and inputs."""
+
+from dataclasses import dataclass
+
+
+@dataclass
+class WorkflowOptions:
+    verbose: bool | None = None
```

Comment:
Just curious, I was using verbose as the profiling flag. Now that it is separated, what do we envision verbose for?

Reply:
Eh, good point, it's not used here yet, and could be feature creep. I'll remove it for now